File: Algorithms/Python/sum_of_digits.py
from __future__ import annotations


def sum_of_digits(n: int) -> int:
    """Returns the sum of the digits of a given integer."""
    return sum(int(i) for i in str(n))


if __name__ == "__main__":
    number = 12345
    print(sum_of_digits(number))

File: Algorithms/Java/SumOfDigits.java
class SumOfDigits {
    public static void main(final String[] args) {
        int n = 123;
        System.out.println(sumOfDigits(n));
    }

    public static int sumOfDigits(final int number) {
        int sum = 0;
        int n = number;

        while (n > 0) {
            sum += n % 10;
            n = n / 10;
        }
        return sum;
    }
}
